### PR TITLE
Issue/7063 thumbnail crash

### LINKED ISSUE #7063: Aztec Thumbnail handling crash
Related to changes in #7057 

```
01-08 11:44:45.680 694-694/? E/StagefrightMetadataRetriever: Unable to instantiate an extractor for 'https://testmzorz2.files.wordpress.com/2018/01/vid_20180108_1008371.mp4'.
01-08 11:44:45.684 8105-10003/org.wordpress.android E/WordPress-MEDIA: The passed video path is invalid: https://testmzorz2.files.wordpress.com/2018/01/vid_20180108_1008371.mp4
01-08 11:44:45.687 8105-10003/org.wordpress.android W/WordPress-MEDIA: Failed to retrieve frame from the passed video path: https://testmzorz2.files.wordpress.com/2018/01/vid_20180108_1008371.mp4
01-08 11:44:45.717 8105-8105/org.wordpress.android D/AndroidRuntime: Shutting down VM
01-08 11:44:45.720 8105-8105/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
                                                                     Process: org.wordpress.android, PID: 8105
                                                                     java.lang.NullPointerException: Attempt to invoke virtual method 'void android.graphics.Bitmap.setDensity(int)' on a null object reference
                                                                         at org.wordpress.android.ui.posts.services.AztecVideoLoader$1.onPostExecute(AztecVideoLoader.java:57)
                                                                         at org.wordpress.android.ui.posts.services.AztecVideoLoader$1.onPostExecute(AztecVideoLoader.java:42)
                                                                         at android.os.AsyncTask.finish(AsyncTask.java:695)
                                                                         at android.os.AsyncTask.-wrap1(Unknown Source:0)
                                                                         at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:712)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:105)
                                                                         at android.os.Looper.loop(Looper.java:164)
                                                                         at android.app.ActivityThread.main(ActivityThread.java:6541)
                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                         at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
01-08 11:44:45.730 919-7973/? W/ActivityManager:   Force finishing activity org.wordpress.android/.ui.posts.EditPostActivity

````

### PR DESCRIPTION
**Pull request by mzorz**
Fixes #7063

cc @daniloercoli

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/posts/services/AztecImageLoader.java [hunk 1]
```diff
@@ -76,12 +78,16 @@ public void onResponse(ImageLoader.ImageContainer response, boolean isImmediate)
                     final String cacheKey = url + maxWidth;
                     // Make sure both width ad height respect the max size for the editor
                     bitmap = ImageUtils.getScaledBitmapAtLongestSide(bitmap, maxWidth);
-                    WordPress.getBitmapCache().putBitmap(cacheKey, bitmap);
-                    // By default, BitmapFactory.decodeFile sets the bitmap's density to the device default so, we need
-                    // to correctly set the input density to 160 ourselves.
-                    bitmap.setDensity(DisplayMetrics.DENSITY_DEFAULT);
-                    BitmapDrawable bitmapDrawable = new BitmapDrawable(context.getResources(), bitmap);
-                    callbacks.onImageLoaded(bitmapDrawable);
+                    if (bitmap != null) {
```
Comment (daniloercoli) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/services/AztecImageLoader.java]: AS says this is always `true`. I've verified and getScaledBitmapAtLongestSide` returns the original bitmap (that is not null) if the resize is not possible.

File: WordPress/src/main/java/org/wordpress/android/ui/posts/services/AztecVideoLoader.java [hunk 2]
```diff
@@ -52,8 +52,13 @@ protected Bitmap doInBackground(Void... params) {
             protected void onPostExecute(Bitmap thumb) {
                 if (thumb == null) {
                     callbacks.onThumbnailFailed();
+                    return;
                 }
                 thumb = ImageUtils.getScaledBitmapAtLongestSide(thumb, maxWidth);
+                if (thumb == null) {
```
Comment (daniloercoli) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/services/AztecVideoLoader.java]: Same as above. The `return` statement on line 55 is enough to fix the problem.

**Comment by mzorz** [comment]
Thanks for the comments @daniloercoli ! Ready for another pass 👍

**Comment by daniloercoli** [comment]
Looks good!! 👍

**Pull request merged**